qute/extensions/windows.py

```diff
@@ -59,12 +59,14 @@ def resizeEvent(self, event):
         self.storeSize()
         super(MemorableWindow, self).resizeEvent(event)
 
-    # --------------------------------------------------------------------------
     def moveEvent(self, event):
```

Comment:
Not a big fan of resize and move events storing the size - this will get triggered during the resize too.